kikit/panelize_ui_sections.py

```diff
@@ -189,6 +192,15 @@ class SList(SectionBase):
     def validate(self, x: str) -> Any:
         return [v.strip() for v in x.split(",")]
 
+class SVec(SectionBase):
+    def validate(self, x: str) -> Any:
+        return VECTOR2I_MM(*json.loads(x))
+
+class SVecList(SList):
+    def validate(self, x: str) -> Any:
+        s = '[' + x + ']'
+        return [VECTOR2I_MM(*v) for v in json.loads(s)]
```

Comment:
This is inconsistent with `SList` - we already specify a list of layers without `[]`